testsuite/support.py

```diff
@@ -146,6 +146,19 @@ def init_tests(pep8style):
     report = pep8style.init_report(TestReport)
     runner = pep8style.input_file
 
+    def set_hang_closing_option(pep8style, codes):
+        """Set hang-closing option if present in the test codes.
+
+        (mutates pep8style)
+
+        Usage in test:
+        #: #E133:3:1 --with-hanging
```

Comment:
Reading it over, I got the command line arg wrong too, I'll update this, should be ``--hang-closing`` not ``--with-hanging``